src/js/components/projects/detail.tsx

```diff
@@ -435,8 +464,7 @@ const ProjectDetail = (
                     <Trans i18nKey="tourTasksList">
                       Select the Tasks tab to see a list of all the work being
                       done on this Project and who is working on it. Tasks
-                      represent small changes to the Project, and may be grouped
-                      with other Tasks in an Epic.
+                      represent small changes to the Project, and may be part of an Epic.
```

Comment:
@SondraE here one of two copy changes we made

Reply:
@SondraE @stacyk I'm not clear on why this is an improvement. It seems like "may be part of an Epic" is so vague that it's not providing useful information -- why is that better than "may be grouped with other Tasks in an Epic"?

Reply:
@jgerigmeyer We thought "Tasks may be grouped with other Tasks in an Epic" made it sound like users could take a bunch of existing Tasks and pull them together into a group. Probably what should happen here is that we don't try to make these wording changes in this PR, but wait and make them as part of this story: https://trello.com/c/th1Pk6wb/322-i-want-to-better-understand-how-projects-epics-and-tasks-are-related-eg-epics-arent-simply-folders-for-tasks

Reply:
Makes sense to me. I'm fine with this for now, and we can revisit in the other story. 👍